src/main/java/flight/reservation/plane/Helicopter.java

```diff
@@ -1,8 +1,8 @@
 package flight.reservation.plane;
 
-public class Helicopter {
-    private final String model;
-    private final int passengerCapacity;
+public class Helicopter extends Plane{
+    //private final String model;
+    //private final int passengerCapacity;
 
     public Helicopter(String model) {
```

Comment:
crewCapacity is not populated. For helicopters, it seems that crewCapacity = 2, hence refactoring here changed behaviour.